src/fides/api/ctl/database/database.py

```diff
@@ -60,6 +83,154 @@ def create_db_if_not_exists(database_url: str) -> None:
         create_database(database_url)
 
 
+async def load_default_policies() -> None:
+    """
+    Checks whether DSR execution policies exist in the database, and
+    inserts them to target a default set of data categories if not.
+    """
+    FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fides_autogenerated_client"
+    FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+    STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
```

Comment:
I'd prefer if we default the erasure policy to `hmac` instead of `string_rewrite`; it's a safer default for someone unclear on what to choose for their erasure strategy

Reply:
fixed